skrifa/src/outline/error.rs

```diff
@@ -31,6 +31,10 @@ pub enum DrawError {
     ToPath(ToPathError),
     /// Error occured when reading font data.
     Read(ReadError),
+    /// HarfBuzz style drawing with hints is not supported
+    ///
+    /// Error rather than silently returning unhinted per f2f discussion.
```

Comment:
Context seems useful but probably shouldn't be part of the doc comment.

Reply:
Now with significantly fewer /'s